dist/backdraft.js

```diff
@@ -1275,8 +1275,19 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var configsArray = this.columnsConfig;
+      var cleanConfigs = this.columnsConfig.map(function(a,b,c){
```

Comment:
what the the parameters a,b,c? can you rename them to something more descriptive?